unlock-app/src/components/interface/layouts/index/NotificationsMenu.tsx

```diff
@@ -46,6 +59,14 @@ export function NotificationsMenu() {
     })
   }
 
+  if (!isLoading && !error && hooks?.data?.length && hooks?.data?.length > 0) {
+    notifications.push({
+      id: '2',
+      content: <Hooks hooks={hooks.data} refetch={refetch} />,
+      timestamp: new Date(),
+    })
+  }
+
```

Comment:
```suggestion
const notifications = useMemo<NotificationProps[]>(() => {
    const items: NotificationProps[] = []

    if (!email && !['/checkout', '/demo'].includes(pathname)) {
      items.push({
        id: '1',
        content: <PromptEmailLink setModalOpen={setShowModal} />,
        timestamp: new Date(),
      })
    }

    if (!isLoading && !error && hooks?.data?.length > 0) {
      items.push({
        id: '2',
        content: <Hooks hooks={hooks.data} refetch={refetch} />,
        timestamp: new Date(),
      })
    }

    return items
  }, [email, pathname, isLoading, error, hooks, refetch])
```

useMemo seems appropriate. There is another if statement filtering the items and adding to array.